generated/nidaqmx/_library_interpreter.py

```diff
@@ -1774,7 +1781,6 @@ def create_task(self, session_name):
         error_code = cfunc(
             session_name, ctypes.byref(task))
         check_for_error(error_code)
-        return task
```

Comment:
Changing to use func.output_parameters removed `return task` because `Function.__init__` filters out task handles:
https://github.com/ni/nidaqmx-python/blob/master/src/codegen/functions/function.py#L35

I think we need to be more specific about what we mean by "output parameters". The Python API, C DLL, and gRPC proto file all have completely different ideas of what "output parameters" means.

Maybe the answer is that the interpreter codegen should continue to use base_parameters and do its own filtering.

Reply:
I have added a separate method to find the output parameters for interpreters and this issue is now fixed